packages/web-twig/src/Resources/components/Grid/stories/GridAlignment.twig

```diff
@@ -0,0 +1,35 @@
+<Grid cols="2">
+    <Grid cols="1" UNSAFE_className="bg-cover">
+        <DocsBox size="small">Stretch (default)<br/>…</DocsBox>
```

Comment:
I think visual tests will tell you this part is different in React/Twig. Saying "default" here is OK ("normal" is not, see my comment in React).